Assets/Scripts/Actors/ActorComponents/PlayerActorStats.cs

```diff
@@ -0,0 +1,89 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+[RequireComponent( typeof(PlayerActorPhysics) )]
+public class PlayerActorStats : ActorComponent {
+
+	[SerializeField] float _startingMaxStamina = 0.0f;
```

Comment:
I'm iffy on which one I prefer. I like 3.0f, 4.0f etc. because it very visually reminds me that it's a floating point / real number.